src/django/contricleaner/lib/context/contri_cleaner_context.py

```diff
@@ -3,15 +3,15 @@
 from contricleaner.lib.dto.row_dto import RowDTO
 from contricleaner.lib.serializers.row_serializers.row_composite_serializer \
     import RowCompositeSerializer
-from contricleaner.lib.source_parser import SourceParser
+from contricleaner.lib.parsers.data_parser import DataParser
 
 
-class ContriCleanerSerializer:
-    def __init__(self, source: SourceParser):
-        self.source = source
+class ContriCleanerContext:
```

Comment:
Again, let's not use context here. this is not a context in fact because it contains some logic.